checkexperiments.py

```diff
@@ -0,0 +1,266 @@
+            try:
+                r.raise_for_status()
+            except requests.HTTPError:
+                return
+            else:
+                local = copy.deepcopy(r.json()['@graph'])
+                graph.extend(local)
+    # checkexperiments using a query
+    else:
+        r = session.get(
+            urljoin(
+                url,
+                '/search/?type=Experiment' \
+                '&format=json&frame=object&limit=all&' + search_query))
+        try:
+            r.raise_for_status()
+        except requests.HTTPError:
+            return
+        else:
+            graph = r.json()['@graph']
+    print('number of experiments: ' + str(len(graph)))
+    for ex in graph:
+        experiment_accession = ex.get('accession')
+        award_request = session.get(urljoin(
+            url,
+            ex.get('award') + '?frame=object&format=json'))
+        award_obj = award_request.json()
+        replicates = set()
+        replicates_reads = {}
+        dates = []
+        files = []
+        try:
+            if ex.get('replicates'):
+                for replicate in ex.get('replicates'):
+                    replicate_request = session.get(urljoin(
+                        url,
+                        replicate + '?frame=object&format=json'))
+                    replicate_obj = replicate_request.json()
+                    if replicate_obj.get('status') not in ['deleted']:
+                        replicates.add(replicate_obj['@id'])
+                        replicates_reads[replicate_obj['@id']] = 0
+                if  ex.get('files'):
+                    for file_acc in ex.get('files'):
+                        file_request = session.get(urljoin(
+                            url,
+                            file_acc + '?frame=object&format=json'))
+                        file_obj = file_request.json()
+                        if file_obj.get('file_format') == 'fastq' and \
+                           file_obj.get('status') not in ['uploading',
+                                                          'content error',
+                                                          'upload failed',
+                                                         ]:
+                            file_date = datetime.datetime.strptime(
+                                file_obj['date_created'][:10], "%Y-%m-%d")
+                            dates.append(file_date)
+                            files.append(file_obj)
+                            if file_obj.get('read_count') and file_obj.get('replicate'):
+                                if not replicates_reads.get(file_obj.get('replicate')) is None:
+                                    replicates_reads[file_obj.get('replicate')] += \
+                                        file_obj.get('read_count')
+        except requests.exceptions.RequestException as e:
+            print (e)
```

Comment:
remove extra space between 'print' and it's argument '(e)'

Reply:
done